example/glue/ScavengerRootScanner.hpp

```diff
@@ -56,6 +58,7 @@ class MM_ScavengerRootScanner : public MM_Base
 	void
 	scavengeRememberedSet(MM_EnvironmentStandard *env)

```

Comment:
Is this flush needed?  I feel like the helper threads and the master thread should not have any remembered set entries that have not been flushed at this point.  Do we not scan the remembered set before any other operations?